src/cleng/node.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include "point.h"
+#include <map>
+#include <string>
+
+class Node {
```

Comment:
You should add a virtual destructor to this class to prevent memory leaks.

Reply:
Sure, thank you!